### ISSUE TITLE
URL in PDF file not clickable

### ISSUE THREAD

**Issue by djermann** [nextcloud/files_pdfviewer#44]
A customer noticed that URLs in PDF files, which get uploaded to Nextcloud, do not get detected automatically, so one can not click on them like a link. This only happens with URLs that are saved as plain text, not as Hyperlinks. Therefore, I want to request this feature.

### Steps to reproduce

Create a PDF file with a URL inside that is not a hyperlink, just plain text.
Upload it to Nextcloud.
Open it in the web interface of Nextcloud.
### Expected behaviour

The URL should be clickable like a link.

### Actual behaviour

The URL is not clickable.

**Comment by premar** [comment]
Hello

I stumbled upon the same problem. I tested with the .pdf provided from @nickvergessen: https://www.antennahouse.com/XSLsample/pdf/sample-link_1.pdf

What i can say that its browser related, and at the moment only IE 11 is not supporting external links with .pdf in Nextcloud. It's working fine with Chrome, Firefox 58.0.2, Edge 38.14393.

I tested with Nextcloud 13.0.4 and 13.0.5, with same results.
